content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -427,8 +427,15 @@ or updating objects that contain Pod templates, such as Deployments, Jobs, State
 See [Enforcing Pod Security at the Namespace Level](/docs/concepts/security/pod-security-admission)
 for more information.
 
-## seccomp.security.alpha.kubernetes.io/pod and container.seccomp.security.alpha.kubernetes.io/[NAME] (deprecated)
+## seccomp.security.alpha.kubernetes.io/pod (dperecated)
 
-The seccomp annotations have been deprecated since Kubernetes v1.19 and will
-become non-functional in v1.25. Please use the `seccompProfile` of the
-`SecurityContext` instead.
\ No newline at end of file
+The seccomp annotations have been deprecated since Kubernetes v1.19 and will become non-functional in v1.25.
+seccomp is now a class inside of the pod or container 
+and can be changed by using `type` of the `seccompProfile` of the `securityContext`. For additinal info
+[visit here](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)
```

Comment:
How about 

> Learn more about [setting the security context for a pod.](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)

Reply:
Should I also include example from the mentioned link in the file or just put a link in the end like it is already done??

Reply:
We generally avoid "visit here" or "click here" in the docs, and instead work to make links descriptive, like my example above.

Feel free to ping me on kubernetes slack for questions :)